src/hb-ot-var-avar-table.hh

```diff
@@ -128,11 +162,24 @@ struct avar
     const SegmentMaps *map = &firstAxisSegmentMaps;
     for (unsigned int i = 0; i < count; i++)
     {
-      coords[i] = map->map (coords[i]);
+//       coords[i] = map->map (coords[i]);
```

Comment:
guess can't reversed easily, test-ot-style test is working without this not sure why the unmap doesn't work however.